Integrations/Gmail/Gmail.py

```diff
@@ -100,6 +120,14 @@ def get_credentials(additional_scopes=None, delegated_user=None):
     return cred.create_delegated(delegated_user)
 
 
+def get_service(serviceName, version, additional_scopes=None, delegated_user=None):
```

Comment:
in python, use snake_case
 ```suggestion
def get_service(service_name, version, additional_scopes=None, delegated_user=None):
```